udata/core/spatial/models.py

```diff
@@ -53,6 +58,24 @@ def __unicode__(self):
     def __html__(self):
         return gettext(self.name) + ' <i>(' + self.code + ')</i>'
 
+    def logo_url(self, external=False):
+        path = ((self.logo.filename
```

Comment:
It doesn't work because my use-case is a bit special: sometimes the path is filed within the DB but the file cannot be download so I need to check the existence directly against the file system…